plugins/item_tasks/server/rest.py

```diff
@@ -21,6 +22,8 @@ def __init__(self):
         self.route('POST', (':id', 'execution'), self.executeTask)
         self.route('POST', (':id', 'slicer_cli_description'), self.runSlicerCliDescription)
         self.route('PUT', (':id', 'slicer_cli_xml'), self.setSpecFromXml)
+        self.route('POST', (':id', 'json_description'), self.runJsonTasksDescription)
+        self.route('PUT', (':id', 'add_json_specs'), self.addJsonSpecs)
```

Comment:
This should probably be a `POST` since it's creating documents rather than updating them, and the path should be `json_specs` instead of `add_json_specs` (we avoid verbs in the path when possible, which is a convention of REST).